packages/gen-ai/frontend/src/app/AIAssets/components/mcp/MCPServersTable.tsx

```diff
@@ -0,0 +1,138 @@
+import * as React from 'react';
+import { useCheckboxTable, Table, DashboardEmptyTableView } from 'mod-arch-shared';
+import { SearchType } from 'mod-arch-shared/dist/components/DashboardSearchField';
+import { MCPServer, MCPServerFromAPI } from '~/app/types';
+import { ServerStatusInfo } from '~/app/hooks/useMCPServers';
+import { useMCPSelectionContext } from '~/app/context/MCPSelectionContext';
+import NoData from '~/app/EmptyStates/NoData';
+import { transformMCPServerData } from '~/app/utilities/mcp';
+import MCPServerTableRow from './MCPServerTableRow';
+import MCPServersToolbar from './MCPServersToolbar';
+import MCPServerColumns from './MCPServerColumns';
+
+interface MCPServersTableProps {
+  /** Array of MCP servers from API */
+  servers: MCPServerFromAPI[];
```

Comment:
Is there another source you're expecting the servers to come from?
```suggestion
  servers: MCPServer[];
```